Engines/Wine/QuickScript/Quick Script/script.js

```diff
@@ -135,6 +135,20 @@ module.default = class QuickScript {
         return this;
     }
 
+    /**
+     * determines which Wine version should be used
+     * required in case the version is computed by a function
+     * @param {wizard} wizard setup wizard (e.g. to show download progress of versions json)
+     * @returns {void}
+     */
+    _determineWineVersion(wizard) {
+        if (this._wineVersion && {}.toString.call(this._wineVersion) === '[object Function]') {
```

Comment:
alternatively `if (this._wineVersion && typeof this._wineVersion === "function")` and `if (this._wineVersion && this._wineVersion instanceof Function)` should also work

Reply:
I've used the accepted answer of this StackOverflow: https://stackoverflow.com/questions/5999998/check-if-a-variable-is-of-function-type. I didn't really check why this is the accepted answer or why the other possibilities have not been accepted, but honestly, I don't see a reason to change it.

Reply:
You can use the solution you have. It is just not that intuitive in my opinion. I honestly don't even know what `{}.toString.call(this._wineVersion)` does...

My alternatives in comparison are easier (I would even say almost trivial) to understand

Reply:
I've changed it.